app/components/favorite-routes/favorite-route-box.tsx

```diff
@@ -65,29 +68,44 @@ const STATION_ORIGIN_CIRCLE: ViewStyle = {
   borderColor: "#fff",
 }
 
+const CONTENT: ViewStyle = {
+  position: "relative",
+}
+
+const LINE_WIDTH = 2.5 * fontScale
+
 const LINE: ViewStyle = {
   position: "absolute",
-  start: 17.3,
-  width: 2.5 * fontScale,
-  height: 23,
+  start: (STATION_CIRCLE_SIZE - LINE_WIDTH) / 2,
+  top: STATION_CIRCLE_SIZE,
+  width: LINE_WIDTH,
+  height: 32,
   backgroundColor: "lightgrey",
   shadowOffset: { height: 0, width: 0 },
   shadowOpacity: 0.5,
   shadowRadius: 4,
   elevation: 2,
   zIndex: 0,
 }
+
+const ROUTE_NAME: TextStyle = {
```

Comment:
I've experimented with increasing the font size a little and I think it looks better, wdyu?

**iOS:**
![Simulator Screen Shot - iPhone 11 - 2021-07-05 at 10 01 17](https://user-images.githubusercontent.com/13344923/124430609-0be05f80-dd78-11eb-96f9-537e44ba636c.png)

**Android:**
![IMAGE 2021-07-05 10:01:42](https://user-images.githubusercontent.com/13344923/124430653-1995e500-dd78-11eb-8c4c-6b6e23dad2a5.jpg)

Style properties I've used:
```typescript
const ROUTE_NAME: TextStyle = {
  marginBottom: spacing[2],
  fontSize: 20,
  fontWeight: "bold",
  color: color.whiteText,
}
```

Reply:
Sure, why not.

One thing though - Is there a reason the app uses Heebo for iOS but Arimo (the default) for Android?

Reply:
The default font for iOS (Arial) didn't do good for the design, and Heebo seemed to be better fit with the OS.
Arimo seemed to blend well with the design and the OS, and a couple of Android users I asked liked the design with Arimo more than Heebo.

Add to that that implementing custom fonts for Android is a bit of a pain (we'll have to use `fontFamily` to change font weights instead of `fontWeight`), I went with the simple way of using the system font. 

Do you find the design looks better with Heebo?

Reply:
Seems legit. I asked because of the inconsistency, but Arimo looks absolutely fine :)